src/main/groovy/org/spongepowered/asm/gradle/plugins/MixinExtension.groovy

```diff
@@ -323,7 +327,7 @@ public class MixinExtension {
         
         AbstractArchiveTask.metaClass.getRefMaps = {
             if (!delegate.ext.has('refMaps')) {
-                delegate.ext.refMaps = project.layout.configurableFiles();
+                delegate.ext.refMaps = project.objects.fileCollection()
```

Comment:
Yes, this is a change in Gradle, but hoonestly because MinxinGradle supports so many versions, and this is groovy, it'd be good to add a runtime test to switch between the proper apis instead of  bumping the build system and breaking backwards compat